capstone-ui/src/Components/StateMap/StateMap.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect } from 'react';
+import * as topojson from 'topojson-client';
+import usData from '../../data/us.json';
+import "./StateMap.css";
+
+export default function StateMap({ stateName }) {
+
+    const FIPSfromStateName = {
```

Comment:
Can we add this to another file to increase readability. Maybe even have this in a json file we read from.